app/Http/Controllers/Order/ExtendedOrderController.php

```diff
@@ -265,6 +281,50 @@ private function expiryTill($expiry, $expiryTill, $join)
         }
     }
 
+    /**
+     * Searches for Subcription From Date.
+     *
+     * @param string $expiry The Subcription From Date
+     * @param object $join
+     *
+     * @return Query
+     */
+    private function subFrom($till, $from, $join)
+    {
+        if ($from) {
+            $fromdate = date_create($from);
+
+            $from = date_format($fromdate, 'Y-m-d H:m:i');
+            $tills = date('Y-m-d H:m:i');
+
+            $tillDate = $this->getTillDate($from, $till, $tills);
+            $join = $join->whereBetween('subscriptions.created_at', [$from, $tillDate]);
+
+            return $join;
+        }
+    }
+
+    /**
+     * Searches for Order Till Date.
+     *
+     * @param string $expiry The Order Till Date
+     * @param object $join
+     *
+     * @return Query
+     */
+    private function subTill($from, $till, $join)
```

Comment:
read the above comments for do the same here

Reply:
Fixed